[cmake] Detect GLU and disable targets using it if GLU is not found

When compiling pcl 1.7.2 on archlinux without glu installed, I get a compilation error that `GL/glu.h` can not be found.  I realise this can be fixed easily by installing glu but it seems to me that cmake should have errored out before compilation even started or disabled whatever features depend on glu.

As far as I can tell, the cmake files look for opengl and optionally glut, but not glu.

```
/build/pcl/src/pcl-pcl-1.7.2/apps/point_cloud_editor/src/cloudEditorWidget.cpp:51:21: fatal error: GL/glu.h: No such file or directory
 # include <GL/glu.h>
```

--- a/apps/point_cloud_editor/CMakeLists.txt
+++ b/apps/point_cloud_editor/CMakeLists.txt
@@ -1,7 +1,7 @@
 set(SUBSUBSYS_NAME point_cloud_editor)
 set(SUBSUBSYS_DESC "Point Cloud Editor - Simple editor for 3D point clouds")
 set(SUBSUBSYS_DEPS common filters io apps)
-set(SUBSUBSYS_EXT_DEPS vtk ${QTX} OpenGL)
+set(SUBSUBSYS_EXT_DEPS vtk ${QTX} OpenGL OpenGL_GLU)
 set(REASON "")
 set(DEFAULT OFF)
 

